### ISSUE TITLE
Better describe diagnostic_describe_all_properties

### ISSUE THREAD

**Issue by dvorapa** [dart-lang/linter#1898]
**Describe the issue**
When running `flutter analyze` I get this warning for example for the following line:
```
final Random random = Random();
```
```
   info • DO reference all public properties in debug methods •
          lib/main.dart:152:16 • diagnostic_describe_all_properties
```

**To Reproduce**
But in the https://dart-lang.github.io/linter/lints/diagnostic_describe_all_properties.html page there is no description how to fix this issue in lines like this.

**Additional context**
Dart 2.7.0, Flutter 1.12.13

**Comment by dvorapa** [comment]
Is this a bug in detection rather than doc issue?

**Comment by pq** [comment]
@jacob314

**Comment by ZaidSA** [comment]
Guys any updates on this? @pq @jacob314?
As a developer we are not sure how to fix this.

**Comment by pq** [comment]
Hi @ZaidSA.  Could you help us reproduce the issue you're seeing?  The example above doesn't have much to go on.

Sorry for the confusion!

**Comment by jacob314** [comment]
Unless you are developing a shared widget component I do not recommend enabling the lint. I suspect `random` was a field in a Widget class. Tying this lint to a quickfix would clarify what the intent is for library authors.
